_*2
Traceback (most recent call last):
  File "/usr/lib/python3.10/idlelib/run.py", line 578, in runcode
    exec(code, self.locals)
  File "<pyshell#3>", line 1, in <module>
NameError: name '_' is not defined
'hello'
'hello'
10+20
30
_*2
60
3+2*4
11
4/2
2.0
4**2
16
for i in range(1,11)
SyntaxError: expected ':'
for i in range(1,11):
    print("7","*",i,"=",7*i)

    
7 * 1 = 7
7 * 2 = 14
7 * 3 = 21
7 * 4 = 28
7 * 5 = 35
7 * 6 = 42
7 * 7 = 49
7 * 8 = 56
7 * 9 = 63
7 * 10 = 70
s="Hello"
s*3
'HelloHelloHello'
s.upper()
'HELLO'
s.lower()
'hello'
s
'Hello'

